docker-compose.yml

```diff
@@ -71,7 +71,7 @@ services:
     extra_hosts:
       - "host.docker.internal:host-gateway"
     environment:
-      QHANA_URL_MAPPING: '{"(?<=^|https?://)(localhost|host.docker.internal):5005": "qhana-plugin-runner:8080", "(?<=^|https?://)localhost": "host.docker.internal"}'
+      LOCALHOST_PROXY_PORTS: ":5005"
```

Comment:
Can caddy do port ranges? It may be a good idea to expose the whole 5000-5009 range. This allows for the future plugin registry and multiple plugin runners with different config to be accessed via localhost.